.golangci.yml

```diff
@@ -35,8 +35,7 @@ linters:
     - bidichk
     - bodyclose
     - containedctx
-    - contextcheck
-    - deadcode
+    # - contextcheck
```

Comment:
https://linear.app/bufbuild/issue/TCN-492/fix-golangci-lint-context-check-issues-in-bufbuildbuf